package.json

```diff
@@ -93,7 +93,6 @@
 		"commander": "2.20.0",
 		"concurrently": "3.5.0",
 		"copy-webpack-plugin": "4.5.2",
-		"core-js": "3.0.1",
```

Comment:
For what reason is this removed? Based on the errors from the failing build, it seems it could help resolve them to keep it present.

Reply:
I assumed it's not necessary :) It turns out I was wrong. I will circle back after my vacation.

Reply:
I added it now to the `@wordpress/babel-preset-default` plugin as it is necessary for tests to work.